server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -202,10 +206,14 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
                 latency_ = (uint32_t)(psi->outputLatency * psi->sampleRate);
         }
 
+        // input_channels is clipped to the number of available audio inputs (above)
+        // output_channels is equal to the requested number of outputs (not clipped)
+        // input_samples and output_samples vectors are resized
+        // to the number of available audio inputs/outputs
```

Comment:
these last two lines are not really necessary, they just restate what the code is already doing.